doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Timers in Node.js
+layout: docs.hbs
+---
+
+# Timers in Node.js and beyond
+
+The Timers module in Node.js contains functions that execute code after a set
+period of time. Timers do not need to be imported via `require()`, since
+all the methods are available globally to emulate the browser JavaScript API.
+To fully understand when timer functions will be executed, it's a good idea to
+read up on the the Node.js
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
+
+## Controlling the Time Continuum with Node.js
+
+The Node.js API provides several ways to of scheduling code to execute at
+some point after the present moment. The functions below may seem familiar,
+since they are available in most browsers, but Node.js actually provides 
+it's own implementation of these methods. Timers integrate very closely 
+with the system, and despite the fact that the API mirrors the browser 
+API, there are some differences in implementation.
+
+### "When I say so" Execution ~ *`setTimeout()`*
+
+`setTimeout()` can be used to schedule code execution after a designated
+amount of milliseconds. This function is also a part of the browser 
+JavaScript API and can be used in the exact same manner.
+
+`setTimeout()` accepts the code to execute as its first argument and the
+millisecond delay defined as a number literal as the second argument. Additional
```

Comment:
Does not need to be a number literal:

`node -e "setTimeout(function(){ console.log('hi') }, new Number(200))"`